x-pack/plugins/security_solution/server/endpoint/services/actions/action_list.ts

```diff
@@ -205,8 +205,19 @@ const getActionDetailsList = async ({
     throw err;
   }
 
-  // return empty details array
-  if (!actionRequests?.body?.hits?.hits) return { actionDetails: [], totalRecords: 0 };
+  if (actionRequests?.statusCode === 404) {
+    const err = new CustomHttpRequestError(
+      'index_not_found_exception',
+      404,
+      new Error('index_not_found_exception')
+    );
+    logger.error(err);
```

Comment:
I don't think we should throw an error if the index doesn't exist.  This is a valid check that we're doing to see if there is any data created yet.

The result right now is that the Kibana logs are filling up with this error which will confuse users (and other developers).

Here's what I see in the logs while there is no index.

<img width="1728" alt="image" src="https://user-images.githubusercontent.com/56395104/191039822-846f8dee-9ba7-4d32-9383-d07eb967ea07.png">

Reply:
Agree, will check how we can avoid that

Reply:
done: https://github.com/elastic/kibana/pull/140942/commits/6c8a244c6cf788172cee42b3b567918feb535396

Reply:
@dasansol92 thanks!  I don't see the error in the logs anymore